.github/workflows/verify_library_generation.yaml

```diff
@@ -40,6 +40,12 @@ jobs:
         eval "$(pyenv init --path)"
         eval "$(pyenv init -)"
         set +ex
+    - name: install utils (macos)
+      if: matrix.os == 'macos-12'
+      shell: bash
+      run: |
+        brew update --preinstall
+        brew install coreutils
```

Comment:
Would you add source code comment what commands are needed from coreutils?

Reply:
Added a small comment indicating that we need `realpath`